config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  get 'recipes/index'
+  get 'recipes/show'
+  devise_for :users
+  root to: "public_recipes#index"
+
+  resources :users do
+    resources :foods
+    resources :recipes, only: [:index, :show, :destroy] do 
+      resources :recipe_foods 
+      resources :shopping_list
+    end
+  end
```

Comment:
- [ ] [optional] when I navigate to the user page I got the following error please make sure to fix that 
<img width="1428" alt="Screenshot 2023-02-10 at 17 34 45" src="https://user-images.githubusercontent.com/61790587/218147818-17234168-8c98-4702-a8c0-b6d37aee651f.png">